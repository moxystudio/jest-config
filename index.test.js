'use strict';

const jestPreset = require('./index');

it('should receive and return options', () => {
    expect(jestPreset({ should: 'work' })).toHaveProperty('should', 'work');
});

it('should match snapshot', () => {
    expect(jestPreset()).toMatchSnapshot();
});
