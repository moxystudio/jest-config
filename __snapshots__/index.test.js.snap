// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "collectCoverageFrom": Array [
    "www/**/*.js",
  ],
  "coveragePathIgnorePatterns": Array [
    "/*.data.js/",
  ],
  "coverageThreshold": Object {
    "global": Object {
      "branches": 80,
      "functions": 80,
      "lines": 80,
      "statements": -10,
    },
  },
  "moduleNameMapper": Object {
    "/.css$": "<PROJECT_ROOT>/node_modules/identity-obj-proxy/src/index.js",
  },
  "setupFiles": Array [
    "./jest.setup.js",
  ],
  "setupFilesAfterEnv": Array [
    "./node_modules/jest-enzyme/lib/index.js",
  ],
  "testPathIgnorePatterns": Array [
    "/*.data.js/",
  ],
  "transform": Object {
    "/.(eot|ttf|woff|woff2|otf)$": "<PROJECT_ROOT>/node_modules/jest-file/index.js",
    "/.(mp3|flac|wav|aac|ogg|oga|mp4|m4a|webm|ogv)$": "<PROJECT_ROOT>/node_modules/jest-file/index.js",
    "/.(png|jpg|jpeg|gif|webp|ico)$": "<PROJECT_ROOT>/node_modules/jest-file/index.js",
    "/.(svg)$": "<PROJECT_ROOT>/node_modules/jest-file/index.js",
    "/.js$": "<PROJECT_ROOT>/node_modules/babel-jest/build/index.js",
  },
}
`;
